### ISSUE TITLE
About hifigan package

### ISSUE THREAD

**Issue by ThanhPham1987** [keonlee9420/Comprehensive-Tacotron2#4]
Hi @keonlee9420 

How to install hifigan package in model.py file

Best regards,
PeterPham

**Comment by guoaoo** [comment]
@ThanhPham1987 
It doesn't need to install, just import

**Comment by keonlee9420** [comment]
Hi @ThanhPham1987 , as @guoaoo said, you don't have to install anything for that. The importing code is already prepared so the only thing you need to do is to unzip the hifigan pre-trained model at [./hifigan/generator_LJSpeech.pth.tar.zip](https://github.com/keonlee9420/Comprehensive-Tacotron2/blob/main/hifigan/generator_LJSpeech.pth.tar.zip) or [./hifigan/generator_universal.pth.tar.zip](https://github.com/keonlee9420/Comprehensive-Tacotron2/blob/main/hifigan/generator_universal.pth.tar.zip) in the same directory.

**Comment by ThanhPham1987** [comment]
> Hi @ThanhPham1987 , as @guoaoo said, you don't have to install anything for that. The importing code is already prepared so the only thing you need to do is to unzip the hifigan pre-trained model at [./hifigan/generator_LJSpeech.pth.tar.zip](https://github.com/keonlee9420/Comprehensive-Tacotron2/blob/main/hifigan/generator_LJSpeech.pth.tar.zip) or [./hifigan/generator_universal.pth.tar.zip](https://github.com/keonlee9420/Comprehensive-Tacotron2/blob/main/hifigan/generator_universal.pth.tar.zip) in the same directory.

**Comment by ThanhPham1987** [comment]
> > Hi @ThanhPham1987 , as @guoaoo said, you don't have to install anything for that. The importing code is already prepared so the only thing you need to do is to unzip the hifigan pre-trained model at [./hifigan/generator_LJSpeech.pth.tar.zip](https://github.com/keonlee9420/Comprehensive-Tacotron2/blob/main/hifigan/generator_LJSpeech.pth.tar.zip) or [./hifigan/generator_universal.pth.tar.zip](https://github.com/keonlee9420/Comprehensive-Tacotron2/blob/main/hifigan/generator_universal.pth.tar.zip) in the same directory.

**Comment by ThanhPham1987** [comment]
> @ThanhPham1987 It doesn't need to install, just import

hi @keonlee9420 . I imported hifigan. But not working
![image](https://user-images.githubusercontent.com/22832922/157162357-2f9a4555-9d16-49d2-bc9f-dca4cee26a7c.png)

**Comment by guoaoo** [comment]
Hi @ThanhPham1987 , it looks like the IDE you are using is Pycharm, maybe trying to change the working path will solve your problem

**Comment by ThanhPham1987** [comment]
Yes, I am using Pycharm. @guoaoo Which IDE are you using for this code?
Best regards,
PeterPham

**Comment by guoaoo** [comment]
@ThanhPham1987 I use Visual Studio Code for python because of its convenient remote function, but its not the important to your problem.


> Hi @ThanhPham1987 , it looks like the IDE you are using is Pycharm, maybe trying to change the working path will solve your problem

Changing the repo's working path will solve your problem.
